community/udc/src/main/java/org/neo4j/ext/udc/impl/DefaultUdcInformationCollector.java

```diff
@@ -274,7 +274,7 @@ private long determineTotalMemory()
         {
             return ((OperatingSystemMXBean) operatingSystemMXBean).getTotalPhysicalMemorySize();
         }
-        catch ( NoClassDefFoundError e )
+        catch ( Throwable e )
```

Comment:
Maybe it would be cleaner to get JVM vendor name and switch on it?
I assume it would be useful to have an enum describing supported vendors but for now using `org.apache.commons.lang3.SystemUtils.JAVA_VENDOR` might be enough.